### ISSUE TITLE
Missing space in global update command output

### ISSUE THREAD

**Issue by douira** [raineorshine/npm-check-updates#719]
- [x] I have searched for [similar issues](https://github.com/raineorshine/npm-check-updates/issues)
- [x] I am using the latest version of `npm-check-updates`
- [x] I am using `node >= 10.17`

---------------------------

## Steps to Reproduce
Run `ncu -g` when there are global packages to update. I'm on macOS Catalina 10.15.6 with Node 12.14.0 and ncu version 7.1.0.

## Current Behavior
Output is missing a space between `install` and the first package name which breaks when the command is copy-pasted.
```
ncu -g
  [====================] 7/7 100%

  generator-code  1.2.12  →  1.2.21 
  npm             6.14.4  →  6.14.7 

  ncu itself cannot upgrade global packages. Run the following to upgrade all global packages: 

  npm -g installgenerator-code@1.2.21 npm@6.14.7
```

## Expected Behavior
There should be a space between `install` and the first package name. It should be `npm -g install package-name@1.0 ...`

**Comment by dxdc** [comment]
https://github.com/raineorshine/npm-check-updates/pull/718

**Comment by douira** [comment]
Oh I guess I only searched the issues and not the PRs. But I guess here you have the issue too then.

**Comment by raineorshine** [comment]
Thanks!

**Comment by raineorshine** [comment]
Published in `v7.1.1`
